app/components/page_settings_summary_component/view.rb

```diff
@@ -35,5 +40,67 @@ def answer_settings
     def show_selection_options
       answer_settings[:selection_options].map { |option| option[:name] }.join(", ")
     end
+
+    def change_answer_type_path
+      if is_new_question?
+        type_of_answer_new_path(form_id: @draft_question.form_id)
+      else
+        type_of_answer_edit_path(form_id: @draft_question.form_id, page_id: @draft_question.page_id)
+      end
+    end
```

Comment:
The similarity of all these path methods makes me think there's probably a way to abstract them into something reusable

Reply:
Should say this is an optional suggestion unless you can think of a neat way to do it!